### PR TITLE
Version 0.0.1 setup

### PR DESCRIPTION
**Pull request by kmcphillips**
@Shopify/shipping 
cc @qq99 

This does nothing so far. It sets up the gem, its gemfile, dependencies, Railtie, test framework, and (most of these files) a dummy rails app to test with.

PR-ing this so the second PR is just functional code, not framework junk. Will comment inline to highlight important areas.

### CODE REVIEW

File: lib/measured/rails/active_record.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+module Measured::Rails
+  extend ActiveSupport::Concern
+
+  module ClassMethods
+    def measured
+
```
Comment (kmcphillips) [inline lib/measured/rails/active_record.rb]: Future implementation will start here.

File: measured-rails.gemspec [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+  spec.add_runtime_dependency "rails", ">= 4.0"
+  spec.add_runtime_dependency "measured", "0.0.1"
+
+  spec.add_development_dependency "bundler", "~> 1.8"
+  spec.add_development_dependency "rake", "~> 10.0"
+  spec.add_development_dependency "minitest", "~> 5.5.1"
+  spec.add_development_dependency "mocha", "~> 1.1.0"
+  spec.add_development_dependency "pry"
+end
```
Comment (kmcphillips) [inline measured-rails.gemspec]: This file has dependencies, versions, etc.

File: test/test_helper.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+require "mocha/setup"
+require "pry"
+
+require File.expand_path("../dummy/config/environment", __FILE__)
+
+ActiveSupport.test_order = :random
+
+class ActiveSupport::TestCase
+
+end
```
Comment (kmcphillips) [inline test/test_helper.rb]: This file loads the gem and the dummy app. No tests yet, but the test suite runs.

```
~/s/measured-rails(version-0.0.1)$ be rake test
/Users/kevin/.rbenv/versions/2.1.1/bin/ruby -I"lib:test:lib/**/*" -I"/Users/kevin/.rbenv/versions/2.1.1/lib/ruby/gems/2.1.0/gems/rake-10.4.2/lib" "/Users/kevin/.rbenv/versions/2.1.1/lib/ruby/gems/2.1.0/gems/rake-10.4.2/lib/rake/rake_test_loader.rb" "test/active_record_test.rb"
Run options: --seed 56261

# Running:

S

Finished in 0.002140s, 467.2897 runs/s, 0.0000 assertions/s.

1 runs, 0 assertions, 0 failures, 0 errors, 1 skips

You have skipped tests. Run with --verbose for details.
```

**Comment by qq99** [comment]
:+1:

**Pull request merged**